### PR TITLE
Patch update SDK Extension AWS to version 1.0.1

### PR DESCRIPTION
**Pull request by NathanielRN**
# Description

After [releasing the opentelemetry-sdk-extension-aws package as 1.0.0](https://pypi.org/project/opentelemetry-sdk-extension-aws/), we should update the development version to be a beta version like we normally do for packages.

This PR also removes the "Beta" classifier for the package.

Follow up to #667

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

# How Has This Been Tested?

N/A

# Does This PR Require a Core Repo Change?

- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
~- [ ] Changelogs have been updated~
~- [ ] Unit tests have been added~
- [x] Documentation has been updated

### CODE REVIEW

**Comment by owais** [comment]
Next intended should be 1.1.0 and in trunk version should be dev so this should be 1.1.0dev0

File: sdk-extension/opentelemetry-sdk-extension-aws/setup.cfg
Comment (lzchen) [inline sdk-extension/opentelemetry-sdk-extension-aws/setup.cfg:26]: Should add the [stable](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/setup.cfg#L26) classifier.

Comment (NathanielRN) [reply sdk-extension/opentelemetry-sdk-extension-aws/setup.cfg:26]: Thanks for that!

**Comment by NathanielRN** [comment]
@owais Thanks for the tip! I wasn't sure what to set for it so thanks for calling that out 🙂 

I actually decided to make this a patch version `1.0.1` so I can set the classifier correctly. That way if people pull in the latest they'll get the right classifier.

I'll follow up with a new PR afterwards to set it to `1.1.0dev0`.

@open-telemetry/opentelemetry-python-contrib-maintainers Once this passes CI and gets merged, could you please push the `opentelemetry-sdk-extension-aws==1.0.1` tag to the `main` branch please? Thank you! 😄 

```
git tag opentelemetry-sdk-extension-aws==1.0.1
git push origin opentelemetry-sdk-extension-aws==1.0.1
```

**Review by lzchen — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**
